src/Features/Core/Portable/QualifyMemberAccess/AbstractQualifyMemberAccessDiagnosticAnalyzer.cs

```diff
@@ -118,6 +118,11 @@ private void AnalyzeOperation(OperationAnalysisContext context, IOperation opera
                 return;
             }
 
+            if (context.Operation.Parent.Kind == OperationKind.PropertySubpattern)
```

Comment:
Would it be possible to add a quick comment explaining this check, similar to the earlier comments in this method?